### PR TITLE
Dynamic URL support for getUserPermissions (ref app)

### PR DESCRIPTION
**Pull request by danielrischar**
Supports dymanic permissions URLs if specified by the manifest.xml.

### CODE REVIEW

**Comment by ChuckFerring-zz** [comment]
Are there similar changes that need to be made for the CLI?

File: reference-app/Opent2tHelper.js [hunk 1]
```diff
@@ -91,32 +91,33 @@ class Opent2tHelper {
         return deferred.promise;
     }
 
-    getUserPermission(url, answers) {
-        var deferred = q.defer();
-        if (url === undefined) {
-            deferred.resolve();
+    getUrl(onboarding, flow, answers) {
+        if (flow.name === 'url') {
+            var replaceVars = this.getReplaceVars(answers);
```
Comment (shalinijoshi19) [inline reference-app/Opent2tHelper.js:96]: >var [](start = 12, length = 3)

let?

**Comment by shalinijoshi19** [comment]
:shipit:

**Review by shalinijoshi19 — APPROVED** [review]

**Comment by ChuckFerring-zz** [comment]
:shipit:

**Pull request merged**